analysis/study_definition_wave1.py

```diff
@@ -96,4 +96,19 @@
             "incidence": 0.01,
         },
     ),
+    # Is there an associated positive test in the 8 weeks before
+    # covid associated death?
+    covid_test_positive_date=patients.with_test_result_in_sgss(
+        pathogen="SARS-CoV-2",
+        test_result="positive",
+        find_first_match_in_period=False,
+        restrict_to_earliest_specimen_date=False,
+        returning="date",
+        date_format="YYYY-MM-DD",
+        on_or_after="died_ons_covid_any_date - 57 days",
```

Comment:
I'm not sure how certain unknown dates are handled in SGSS -- if for example there are dates expressed as `9999-01-01`, then `between` might be better than `on_or_after`, with the end date maybe death + a few days? 

It's not an essential fix